From d6e3623ab6bfc9b0c261f98a8a89277531c9e63d Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Thu, 22 Jun 2023 08:47:04 -0400
Subject: [PATCH 1920/1936] CMake: Avoid overwriting CMAKE_MODULE_PATH var

Fix(?) the windows build as required find modules like
FindFFTW3F.cmake were not available to cmake.
---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8a9911a1..5a06628b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -19,7 +19,7 @@ find_package(Sndio)
 find_package(FLTK)
 find_package(OpenGL) #for FLTK
 find_package(ECM)
-set(CMAKE_MODULE_PATH ${ECM_FIND_MODULE_DIR})
+list(APPEND CMAKE_MODULE_PATH ${ECM_FIND_MODULE_DIR})
 find_package(LibGit2)
 # lash
 if(PKG_CONFIG_FOUND AND NOT (${CMAKE_SYSTEM_NAME} STREQUAL "Windows"))
-- 
2.47.0

